libr/main/radare2.c

```diff
@@ -391,7 +391,17 @@ R_API int r_main_radare2(int argc, const char **argv) {
 	if (e) {
 		RStrBuf *sb = r_strbuf_new (NULL);
 		while (e && *e) {
-			r_strbuf_appendf (sb, "setenv=%s\n", *e);
+			char *k = strdup (*e);
```

Comment:
may be worth moving this logic into a separate function

Reply:
I have moved it to a new function called r_run_get_environ_profile